### PR TITLE
Remove obsolete folders, and update README for new subfolders.

### PR DESCRIPTION
**Pull request by JuneZhou19**
@XiaowenJiang @chenge3 @turtle-fly @fub2 @xiar

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,14 +2,14 @@ The scripts in different folders are used to quickly build or manage the virtual
...
+data_generater         | Script to generate the fru and sensor data for the new node simulation.
+deb_builder            | Scripts to build openipmi and qemu deb packages.
+diag_arp_flux          | Utility to fix ARP flux.
+dmidecode-2.12         | Utility to collect smbios data.
+docker                 | Contents to enable infrasim-compute to run in a docker container.
+fru_data_parser        | Lib to read and write emu file, aka, ipmi\_sim\_cmd set. Enables to operate FRU data command in human readable means.
 monorailtest           | Refer to the README.md in monorailtest folder.
-ova_builder            | Script to quickly build an OVA image.
+packer                 | Templates and related scripts that used for packer to build images.
 smbiostool             | Script to capture smbios data.
```
Comment (turtle-fly) [inline README.md]: Purpose of smbiostool is to change certain attributes in smbios file, not to capture smbios data. Please update @JuneZhou19

Comment (JuneZhou19) [inline README.md]: @turtle-fly Good information. Description updated.

**Comment by JuneZhou19** [comment]
@fub2 @turtle-fly @xiar @XiaowenJiang @chenge3 
Please review.

File: docker/README.md [hunk 2]
```diff
@@ -5,9 +5,11 @@ Which can make infrasim-compute run in a docker container.
...
 could call on the command line to assemble an infrasim-compute image. Using *docker build*
 users can create an automated build that executes several command-line instructions.
-Command example:
-
-    docker build -t infrasim-compute .
+Steps:
+	1. Copy the Dockerfile from the tools/docker directory into the infrasim-compute directory.
+	2. Run docker build command from infrasim-compute directory. 
+	   Command example:  
+		docker build -t infrasim-compute .
```
Comment (fub2) [inline docker/README.md]: :+1:

**Comment by fub2** [comment]
+1

**Pull request merged**